Server/mods/deathmatch/logic/CDatabaseJobQueue.cpp

```diff
@@ -293,13 +296,6 @@ void CDatabaseJobQueueImpl::UpdateDebugData()
 
     shared.m_Mutex.Lock();
 
-    // Log to console if connection count is creeping up

```

Comment:
This has been removed and not reimplemented in the new system. I assume this is intended as it's no longer a bottleneck.

Is there other dead code or no-longer-useful stuff you can rip out too, such as `m_uiConnectionCountWarnThresh`?

Reply:
@pentaflops what about this one?

Reply:
The message is to help scripters who inadvertanly call dbConnect multiple times. I think it should not be removed